### PR TITLE
chore: Create handbook page for daily updates

### PR DESCRIPTION
**Pull request by andrablaj**
# Description

Create handbook page for Product Team daily updates.

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

File: content/en/contribute/medic/onboarding/daily-updates.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+A: If they tag you or you want to reply to something specific about their updates, it is highly encouraged to do so! You can also react to their messages with an emoji if you want to.
+
+### Q: What sort of feedback should I expect on my updates?
+A: Managers should provide direct, regular, both positive and constructive and helpful feedback on status updates provided by the team members.
+
+### Q: What if I'm alone in the product social time? 
+A: You don’t need to wait for more than 5 or 10 minutes if no one else shows up.
+
+### Q: How frequently should I go to the product social time?
+A: You are not required to join this call, but it is highly encouraged as we’d like teammates to connect not only via Slack or GitHub.
```
Comment (latin-panda) [inline content/en/contribute/medic/onboarding/daily-updates.md:93]: Please move these 2 questions to the team-meetings page.

**Review by latin-panda — CHANGES_REQUESTED** [review]
Thanks @andrablaj !! I left some comments let me know what you think!

File: content/en/contribute/medic/onboarding/daily-updates.md (hunk 1 above)
Comment (andrablaj) [reply content/en/contribute/medic/onboarding/daily-updates.md]: Oh, my bad, thanks for flagging!

File: content/en/contribute/medic/onboarding/daily-updates.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+---
+
+## What
+In our fully remote environment, staying connected and informed is crucial. One of the ways we can ensure transparency, and collaboration, and keep everyone in the loop is by utilizing the #dailies channel in Slack for sharing daily updates.  
+
+## Who
+All the Product Team members are expected to share daily updates in Slack (#dailies channel). Due to the nature of their work, Leadership Product colleagues might share less regular, but more consolidated updates. 
+
+## Why are daily updates essential? 
+Here's why it's relevant:
```
[the remark this answers is not in the document]
Comment (andrablaj) [reply content/en/contribute/medic/onboarding/daily-updates.md:16]: I linked the core competencies below in the Q&A section. Do you have a specific additional place where you would suggest linking them?

File: content/en/contribute/medic/onboarding/team-meetings.md
[the remark this answers is not in the document]
Comment (andrablaj) [reply content/en/contribute/medic/onboarding/team-meetings.md:1]: Excellent suggestion!

File: content/en/contribute/medic/onboarding/daily-updates.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (latin-panda) [reply content/en/contribute/medic/onboarding/daily-updates.md:16]: I think that's enough 👍

**Review by latin-panda — APPROVED** [review]
Glowing work! Thanks!

**Comment by jonathanvq** [comment]
I'm sorry that I could not look at this yesterday. It looks great, @andrablaj , thanks for putting it together in the docs site and @latin-panda for the review.

**Pull request merged**